triage/2022-05-24.md

```diff
@@ -0,0 +1,157 @@
+# 2022-05-24 Triage Log
+
+Overall a positive week for non-incremental performance (roughly 0.5% faster),
+with >1% regressions on incremental benchmarks, primarily due to
+[#95563](https://github.com/rust-lang/rust/pull/95563), which will hopefully be
+investigated in the coming weeks.
+
+Triage done by **@simulacrum**.
+Revision range: [7355d971a954ed63293e4191f6677f60c1bc07d9..43d9f3859e0204e764161ee085a360274b5f3e9a](https://perf.rust-lang.org/?start=7355d971a954ed63293e4191f6677f60c1bc07d9&end=43d9f3859e0204e764161ee085a360274b5f3e9a&absolute=false&stat=instructions%3Au)
+
```

Comment:
FWIW you could run the triage builder again and just copy the summary table over.

Reply:
I might do this later today, let's hold off on merging for now.